scenarioo-client/app/manage/comparisons/createComparisonModal.controller.ts

```diff
@@ -200,6 +203,9 @@ function CreateComparisonModalController($uibModalInstance, BranchesAndBuildsSer
 
     function onSuccessCreation() {
         $uibModalInstance.close();
+        if (vm.onComparisonsPage) {
+            vm.route.reload();
```

Comment:
will this do a browser refresh? Do we need that?

Reply:
Yes. This will refresh the page. This is needed to solve issue 8.1 "After submitting a comparison using "Create Comparison", the Comparisons list should be refreshed (only when the user is on that tab)."

Reply:
ok, I don't really understand why we can't just reload the data but need a page refresh since we are in a single page app.

Reply:
I see your point. The difficulty I couldn't overcome was that we are here in the CreateComparisonModalController whereas the data would need to be reloaded in ComparisonsController. I couldn't find an example where we call one controller from another controller. Do you know how to do that?
As an aside, the refresh function in ComparisonsController also triggers a $route.reload(). The reason why it makes sense there (in my opinion) is that if we only load the data and the data hasn't changed, then there is no apparent change in the UI and the user might think that the refresh button didn't do anything and is broken, whereas if we reload the page then there is a visual feedback that something happened.

Reply:
I think we should instead just change the route to go to the comparison view in any case after creating a new comparison. will try to do it on a separate branch. is ok for now.,